### ISSUE TITLE
Training for body parts

### ISSUE THREAD

**Issue by atique81** [Russell91/TensorBox#151]
Hi @Russell91 ,

I want to use your network to train for multiple body parts (e.g., head, torso etc.). For that, I believe, I just need to have a separate LSTM chain, one for each body part and then combine the loss of all the parts together. Could you please share your thoughts on that?

**Comment by Russell91** [comment]
Yes, you could do this. I think some people at MPI actually had a paper on this sort of idea. I think there are better ways of increasing performance than this sort of specialty trick though.

**Comment by atique81** [comment]
Thanks much for your feedback. I highly appreciate your response on this.
